docker/Dockerfile_mitaka

```diff
@@ -0,0 +1,105 @@
+FROM centos:centos7
```

Comment:
Temp file. To be deleted.

Reply:
fixed in 6b7b3d5a8aa2dfa47964b9ebf0fa5686b0d98ea0